src/components/About.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from "react";
+
+class About extends Component {
```

Comment:
this component doesn't deal with any state so just make it a stateless component. it doesn't need to be a class